### PR TITLE
Require recipient Telegram and validate bot token

### PR DESCRIPTION
**Pull request by garanda21**
This request fix `count` against `null` object.

As a bonus I've added a validation in case the `TELEGRAM_BOT_TOKEN` is empty or missing, with this we avoid error `404 Chat not found`

`php artisan config:cache` is necessary becase new files were created at `/config` folder

### CODE REVIEW

File: app/Filament/Pages/Settings/NotificationPage.php [hunk 1]
```diff
@@ -214,21 +214,30 @@ public function sendTestTelegramNotification()
     {
         $notificationSettings = new (NotificationSettings::class);
 
-        if (count($notificationSettings->telegram_recipients)) {
-            foreach ($notificationSettings->telegram_recipients as $recipient) {
-                \Illuminate\Support\Facades\Notification::route('telegram_chat_id', $recipient['telegram_chat_id'])
-                ->notify(new TelegramNotification('Test notification channel *telegram*'));
+        $bot = \Config::get('telegram.bot');
```
Comment (alexjustesen) [inline app/Filament/Pages/Settings/NotificationPage.php:217]: So the best way to reference config values is through the helper method `config()`, in this case if you want to call an item in the `bot` array the syntax is `config('filename.array.key`) or `config('telegram.bot.token')`.

File: app/Filament/Pages/Settings/NotificationPage.php [hunk 2]
```diff
@@ -214,21 +214,30 @@ public function sendTestTelegramNotification()
...
+
+        if (! empty($bot)) {
```
Comment (alexjustesen) [inline app/Filament/Pages/Settings/NotificationPage.php:219]: For this we should use "return early" should there be a missing token and use the [blank](https://laravel.com/docs/9.x/helpers#method-blank) helper. Example below...

```php
if (blank($bot)) {
    Notification::make()
        ->title('First you need to add \'TELEGRAM_BOT_TOKEN\' on your .env file')
        ->warning()
        ->send();
}
```

After that we can then go into the logic for checking for telegram recipients. 
```php
if (! empty($notificationSettings->telegram_recipients)) {
    foreach ($notificationSettings->telegram_recipients as $recipient) {
         \Illuminate\Support\Facades\Notification::route('telegram_chat_id', $recipient['telegram_chat_id'])
                    ->notify(new TelegramNotification('Test notification channel *telegram*'));
         }

         Notification::make()
             ->title('Test Telegram notification sent.')
             ->success()
             ->send();
} else {
    Notification::make()
        ->title('You need to add recipients to receive Telegram notifications.')
        ->warning()
        ->send();
}
```

**Review by alexjustesen — CHANGES_REQUESTED** [review]
Good catch on the validation, couple of things to adjust and make sure to run `pint` so code quality passes.

**Comment by garanda21** [comment]
> Good catch on the validation, couple of things to adjust and make sure to run `pint` so code quality passes.

Submit new commit with changes! Let me know any other issues...

**Review by alexjustesen — APPROVED** [review]

**Review by alexjustesen — APPROVED** [review]

**Pull request merged**